### ISSUE TITLE
rcov doesn't report coverage when calling a method from the result of a block

### ISSUE THREAD

**Issue by dalmirdasilva** [relevance/rcov#91]
The first image is showing the problem and the second the workaround.

Problem: 
http://www.flickr.com/photos/75749300@N08/6801135929/

Workaround: 
http://www.flickr.com/photos/75749300@N08/6801135919/

**Comment by abedra** [comment]
Please include your version and patch level of Ruby, Your operating system, and version of gcc.

**Comment by dalmirdasilva** [comment]
ok.
- Ruby:
  ruby 1.8.7 (2010-08-16 patchlevel 302) [x86_64-linux]
- OS: 
  Linux mu-station 2.6.35-22-generic #33-Ubuntu SMP Sun Sep 19 20:32:27 UTC 2010 x86_64 GNU/Linux
- gcc -v: 
  Target: x86_64-linux-gnu
  Configured with: ../src/configure -v --with-pkgversion='Ubuntu/Linaro 4.4.4-14ubuntu5' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.4 --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --disable-werror --with-arch-32=i686 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
  Thread model: posix
  gcc version 4.4.5 (Ubuntu/Linaro 4.4.4-14ubuntu5) 

Thank you for the fast replay!

**Comment by abedra** [comment]
Ok thank you. This issue is most likely part of the multiline issues that already exist here in the issues. The coverage processor has quite a few hiccups in it.

**Comment by dalmirdasilva** [comment]
Right, btw, i've read here http://www.aaronbedra.com/2009/10/16/nocov.html (I suppose it is your own site) a good way to 'cov off' some pieces of code that have some coverage issue. is that feature released?

**Comment by abedra** [comment]
Yes it is.
